rpc/README.md

```diff
@@ -417,6 +426,7 @@ http://localhost:8114
         {
             "block_hash": "0xf293d02ce5e101b160912aaf15b1b87517b7a6d572c13af9ae4101c1143b22ad",
             "capacity": "0x2ca86f2642",
+            "cellbase_maturity": false,
```

Comment:
It's better to add a new RPC such as `get_cells_by_lock_hash_v2` instead of breaking the existing one.

Reply:
Creating a new RPC name for compatibility upgrade is not common practice, I would like to follow other blockchains (btc/eth)

Reply:
I incline to add new RPC, move the old one to Deprecated module